onnxruntime/test/testdata/test_data_generation/lr_scheduler/warmup_linear_scheduler_warmupstep-70_restored.json

```diff
@@ -0,0 +1,296 @@
+        [
+            0.0009000000000000001
+        ]
+    ],
+    [
+        64,
+        [
+            0.0009142857142857143
+        ]
+    ],
+    [
+        65,
+        [
+            0.0009285714285714287
+        ]
+    ],
+    [
+        66,
+        [
+            0.0009428571428571429
+        ]
+    ],
+    [
+        67,
+        [
+            0.0009571428571428573
+        ]
+    ],
+    [
+        68,
+        [
+            0.0009714285714285714
+        ]
+    ],
+    [
+        69,
+        [
+            0.0009857142857142857
+        ]
+    ],
+    [
+        70,
+        [
+            0.001
+        ]
+    ],
+    [
+        71,
+        [
+            0.0009666666666666667
+        ]
+    ],
+    [
+        72,
+        [
+            0.0009333333333333333
+        ]
+    ],
+    [
+        73,
+        [
+            0.0009000000000000001
+        ]
+    ],
+    [
+        74,
+        [
+            0.0008666666666666667
+        ]
+    ],
+    [
+        75,
+        [
+            0.0008333333333333334
+        ]
+    ],
+    [
+        76,
+        [
+            0.0008
+        ]
+    ],
+    [
+        77,
+        [
+            0.0007666666666666667
+        ]
+    ],
+    [
+        78,
+        [
+            0.0007333333333333333
+        ]
+    ],
+    [
+        79,
+        [
+            0.0007
+        ]
+    ],
+    [
+        80,
+        [
+            0.0006666666666666666
+        ]
+    ],
+    [
+        81,
+        [
+            0.0006333333333333333
+        ]
+    ],
+    [
+        82,
+        [
+            0.0006
+        ]
+    ],
+    [
+        83,
+        [
+            0.0005666666666666667
+        ]
+    ],
+    [
+        84,
+        [
+            0.0005333333333333334
+        ]
+    ],
+    [
+        85,
+        [
+            0.0005
+        ]
+    ],
+    [
+        86,
+        [
+            0.00046666666666666666
+        ]
+    ],
+    [
+        87,
+        [
+            0.00043333333333333337
+        ]
+    ],
+    [
+        88,
+        [
+            0.0004
+        ]
+    ],
+    [
+        89,
+        [
+            0.00036666666666666667
+        ]
+    ],
+    [
+        90,
+        [
+            0.0003333333333333333
+        ]
+    ],
+    [
+        91,
+        [
+            0.0003
+        ]
+    ],
+    [
+        92,
+        [
+            0.0002666666666666667
+        ]
+    ],
+    [
+        93,
+        [
+            0.00023333333333333333
+        ]
+    ],
+    [
+        94,
+        [
+            0.0002
+        ]
+    ],
+    [
+        95,
+        [
+            0.00016666666666666666
+        ]
+    ],
+    [
+        96,
+        [
+            0.00013333333333333334
+        ]
+    ],
+    [
+        97,
+        [
+            0.0001
+        ]
+    ],
+    [
+        98,
+        [
+            6.666666666666667e-05
+        ]
+    ],
+    [
+        99,
+        [
+            3.3333333333333335e-05
+        ]
+    ]
+]
```

Comment:
newline.

Reply:
this is auto generated by json.dump. I think it is fine. :)